test/dialects/snowflake_test.py

```diff
@@ -17,6 +17,7 @@
             "create table orders_clone_restore clone orders at (timestamp => "
             "to_timestamp_tz('04/05/2013 01:02:03', 'mm/dd/yyyy hh24:mi:ss'));",
         ),
+        ("SelectStatementSegment", "SELECT issue FROM issue;"),
```

Comment:
Could you move this test to one of the select statement files in `test/fixtures/dialects/snowflake`?

We try to avoid python tests unless really necessary as SQL tests are 1) Easier to write, 2) Allow us to see the parse tree YML by automatically generating that allowing us to track any changes in that in future PRs.

More info here: https://github.com/sqlfluff/sqlfluff/wiki/Contributing-Dialect-Changes#adding-test-cases-for-your-changes